lib/adapter.js

```diff
@@ -6903,6 +6903,11 @@ function Adapter(options) {
 
         this._addAliasSubscribe = (aliasObj, pattern, callback) => {
             if (aliasObj && aliasObj.common && aliasObj.common.alias && aliasObj.common.alias.id) {
+                if (aliasObj.type !== 'state') {
+                    logger.warn(`${this.namespaceLog} Expected alias ${aliasObj._id} to be of type "state", got "${aliasObj.type}"`);
+                    return typeof callback === 'function' && setImmediate(() => callback(`Expected alias ${aliasObj._id} to be of type "state", got "${aliasObj.type}"`));
```

Comment:
@AlCalzone ALso Error object here or ?!

Reply:
Okay, I wanted to prevent bigger crashes because of this in our current stable candidate.

Reply:
Some such changes were added by code police PR ... so I wanted at least to discuss it :-)